### DISCUSSION TITLE
Which disk-fs is the best option for volumes?

### DISCUSSION THREAD

**Discussion by peedy2495** [seaweedfs/seaweedfs#2675] [General]
As asked in the headline ... has anyone skills about?

**Comment by chrislusf** [comment]
To get any answer, please first clarify that "best" is for reliability, or simplicity, or throughput, or latency, or large files, or small files, or repeated reads, or random reads, or append only, or random writes, ...?

**Comment by flomo29** [comment]
Would you prefer XFS or BTRFS? Btrfs has checksum, compression and one can use external deduplication progs. But XFS has only checksum for metadata. But you said that the Seaweed Volumes have checksums and compression too. So does it make sense to choose BTRFS because of the additional features or is that overkill and we can use XFS?

**Reply by chrislusf** [reply]
Does not matter much. I do not see any report about which FS is better. Hardware speed matters more.
